data/status/status_022.xml

```diff
@@ -15,11 +15,12 @@
 
 <history>
 
+  <history-entry date='July 24, 2021'>Language identified as Gelao via Omniglot, assigned ISO 639-3 code 'giw' for White Gelao. Thanks to Adam Twardoch</history-entry>
```

Comment:
I don't think there is enough evidence to support that identification. Gelao [gio] was split in [gqu], [aou], [giq], [gir] and [giw], and this could be any of those.